source/_docs/custom-https.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Configure Custom HTTPS Certificate with Stacked CDNs
+description: Learn how to configure a third-party CDN provider with Pantheon's Global CDN so you can terminate HTTPS using a custom certificate.
+tags: [cacheedge]
+---
+The following information pertains only to sites that require an additional CDN service, for the primary use case of running a custom certificate to terminate HTTPS instead of [our free, automated HTTPS](/docs/https/). While Pantheon's certificate is shared amongst other domains, it is entirely secure; concerns regarding the legitimacy are entirely cosmetic.
+
+Pantheon does not support using a custom certificate. Sites that require a custom certificate (e.g., extended validation required) must utilize a third-party service provider to do so.
+
+## Supported Providers
```

Comment:
Let's just start with:
- CDN (name)
- Works with Global CDN (Yes|No|Unknown) .  - Yes could link to our doc/section with details?
- Maybe also a column like: Notes/Caveat/Special considerations? Not sure we need that one yet. Maybe start with the first two columns?